<launch>
  <!-- Parameter Server from yaml config files-->
  <rosparam file="$(find pongbot)/config/pongbot_config.yaml" />
  <arg name="namespace"               default="pongbot"/>

  <node pkg="pongbot" name="pongbot_driver" type="pongbot_driver"
    output="screen"/>

</launch>